### ISSUE TITLE
plan to issue a fuzz driver to OSS-Fuzz

### ISSUE THREAD

**Issue by secsys-go** [rclone/rclone#7315]
We are developing a new technique to automatically transform your official unit tests into fuzz drivers. And our auto-generated fuzz driver successfully found your project's bug #6310 . Now we plan to submit this fuzz driver to [OSS-Fuzz Project](https://github.com/google/oss-fuzz). We hope to obtain your approval. Thanks a lot.

We are sorry to submit the PR #7283 without further check. We have carefully reviewed this version and fixed some errors. The fuzz driver's source code is followed. We are appreciated if you could point out possible errors we have not found in this version and provide us with some suggestions to improve our fuzz driver's quality.

```go
package serve

import (
         "github.com/rclone/rclone/fstest/mockobject"
)
import (
         "net/http/httptest"
)
import (
         "testing"
)

func FuzzTestObjectBadRange(f *testing.F) {
	f.Add("aFile", "0123456789")
	f.Fuzz(func(t *testing.T, data1 string, data2 string) {
            w := httptest.NewRecorder()
            r := httptest.NewRequest("GET", "http://example.com/aFile", nil)
            r.Header.Add("Range", "xxxbytes=3-5")
            o := mockobject.New(data1).WithContent([]byte(data2), mockobject.SeekModeNone)
            Object(w, r, o)
            w.Result()
	})
}
```

**Comment by animosity22** [comment]
Are you submitting a new PR? Wasn't sure what this issue was for.

**Comment by secsys-go** [comment]
No, now we just have a proposal. We are security research group and want to use fuzz test to enhance the security of open source project. We are developing a new technique to automatically transform your official unit tests into fuzz drivers. This fuzz driver is auto-generated by our new technique.

[OSS-Fuzz](https://github.com/google/oss-fuzz) is a project that is used to continuously fuzz open source software, which is supported by Google. We want to submit this fuzz driver to OSS-Fuzz, but their maintainers require your approval before they will accept our fuzz driver.

We hope to get the following supports from you:
* Help us review this fuzz driver. If there are fixes needed, we will do some modifications.
* If you agree that we submit the fuzz driver to OSS-Fuzz, we want to obtain your official approval. And then we will submit PR to OSS-Fuzz.

Thanks a lot.
